docs/developer/how_to_contribute.md

```diff
@@ -34,9 +36,13 @@ The following directories also exist but are expected to be temporary:
 1. Clone the repository locally.
 1. Install dependencies with `npm install`.
 1. Copy the `.env.default` file to `.env`, and replace the values that are marked `TODO`
-    1. `CLERK_API_KEY` - Get from `@dvargas92495`
-1. Ensure you have a local instance of `mysql` running, with a user with username and password both with the value `samepage_network`, running on port 3306, and a database created called `mysql`.
-1. Apply the SamePage schema to your local `mysql` instance by running `npx ts-node scripts/cli.ts plan --sql`, followed by `npx ts-node scripts/cli.ts apply --sql`.
+   1. `CLERK_SECRET_KEY` - Get from `@dvargas92495`
+   2. `STRIPE_SECRET_KEY` - Get from `@dvargas92495`
+1. Ensure you have a local instance of `mysql` running, with a user with username and password both with the value `samepage_network`, running on port 3306, and a database created called `samepage_network`.
+   1. Create Database: `CREATE DATABASE samepage_network;`
+   2. Create User: `CREATE USER 'samepage_network'@'localhost' IDENTIFIED BY 'samepage_network';`
+   3. Grant Privileges: `GRANT ALL PRIVILEGES ON samepage_network.* TO 'samepage_network'@'localhost';`
+1. Apply the SamePage schema to your local `mysql` instance by running `npx ts-node scripts/cli.ts plan --sql`, followed by `npx ts-node scripts/cli.ts apply --sql --bare`.
```

Comment:
Let's say `SamePage database schema`

There are tons of different schemas in the world of SamePage so it's good to clarify whenever we can